src/app/pages/careers/frontend/prysmatic-careers-frontend.page.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component } from '@angular/core';
+import { JobPost } from '../../../models/job-post';
+
+@Component({
+  selector: 'prysmatic-careers-sdet',
```

Comment:
is this selector right? shouldn't it be frontend?

Reply:
ty yep will change

Reply:
```suggestion
  selector: 'prysmatic-careers-frontend',
```